src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+body,
+html {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
   background-image: url(./images/star-active.svg);
}
```

You have one property which duplicated. 
You can unite selectors, and write one property as I see.
Try it. Thanks!